crt_portal/cts_forms/templates/forms/word_counter.html

```diff
@@ -0,0 +1,19 @@
+<noscript>
```

Comment:
I don't think it is needed for this PR, but once we have a few of these we can put them in a folder. I like to call these "snippets" but, we can call them something better if you like.

Reply:
Sounds good!